jenkinsfiles/cordova_jenkinsfile.groovy

```diff
@@ -1,14 +1,55 @@
 /**
 * Cordova Jenkinsfile
 */
-node('android') {
-   stage 'Checkout'
-	 checkout scm
 
-	 stage 'Prepare'
-	 cordova platform add android
-	 cordova prepare
+def platform = params?.PLATFORM?.trim()
+def buildType = params?.BUILD_CONFIG?.trim()
+def keystore_path = "{env.ANDROID_HOME}/android.debug"
 
-	 stage 'Build'
-	 cordova build android
-}
+node(platform) {
+    stage("Checkout") {
+        checkout scm
+    }
+
+    stage("Prepare") {
+        sh "cordova platform add $platform"
+        sh "cordova prepare $platform"
+        if(platform == 'android')
+	        if (buildType == 'debug') {
+	            file_content= sh (
+	                script: "jq --arg test_path '${keystore_path}' \'.android.debug.keystore = $test_path\' -r build.json",
+	                returnStdout: true
+	                )
+	        	sh "echo '${file_content}' > build.json"
+	        }else{
+	            file_content= sh (
+	                script: "jq --arg test_path '${keystore_path}' \'.android.release.keystore = $test_path\' -r build.json",
+	                returnStdout: true
+	                )
+		        sh "echo '${file_content}' > build.json"
```

Comment:
I think it will generate an unsigned apk if we run "cordova build android --release"